README.md

```diff
@@ -62,18 +62,23 @@ Host: <host ip> # default: 0.0.0.0
 Port: <host port> # default: 8089
 
 [Microsoft Teams]
```

Comment:
IMHO you should specify that keys not followed by an integer number are still supported

Reply:
Changed variable names to "Connector1" and "ConnectorTwo", I guess it seems clear now :)